include/qb/qblog.h

```diff
@@ -113,13 +118,18 @@ extern "C" {
  *	qb_log_ctl(mytarget, QB_LOG_CONF_THREADED, QB_TRUE);
  * @endcode
  *
- * To workaround your syslog daemon filtering all messages > LOG_INFO
+ * To workaround your syslog daemon filtering messages with the priority of
+ * @c LOG_DEBUG or of lesser importance while such messages should still get
+ * across without any system-wide changes (here, proportionally tweaking the
+ * priority of these such that @c LOG_DEBUG gets promoted to @c LOG_INFO):
  * @code
```

Comment:
I think the original message is fine here. The comment here is too wordy and complex and reads very badly in English.

Reply:
So it was just me, who wasn't able to decipher what exactly to expect
from using that configuration toggle?  I'd doubt that.  Are you able
to tell from the original description, without looking into source,
and forgetting my rewording?

Reply:
IT seems pretty clear what happens - even if WHY it happens is a total mystery! If you can come up with something clearer I'll happily include it. But I'm afraid that's incomprehensible English